### PR TITLE
Fix: Backend must define serialization for SFRs

### PR DESCRIPTION
**Pull request by dwhswenson**
Fixes issues from behind-the-scenes changes introduced by https://github.com/openpathsampling/openpathsampling/pull/1035.

This issue makes me think that the current approach for registering serialization types isn't the most natural, but fixes on that can come another time.

### CODE REVIEW

File: tests/09_storable_functions.ipynb
Comment (sroet) [inline tests/09_storable_functions.ipynb:247]: Is this the only code change (it was the only one I could spot in-between all the output changes), if so, this LGTM

**Review by sroet — APPROVED** [review]
This LGTM (assuming I identified the only part of the code that changed). Also, just letting you know that the write access  for `openpathsampling` has not carried over to this repo

File: tests/09_storable_functions.ipynb
Comment (dwhswenson) [reply tests/09_storable_functions.ipynb:247]: Yep, that's it. This is done internally in `StorageFunctionHandler.register_function`, although I'm not sure that's the best place for it, in terms of level of abstraction. It seems like maybe this should be handled within the backend, e.g., as part of `backend.register_storable_function`. But I think there was some reason I couldn't do that (or maybe doing that would make other things messy).

https://github.com/openpathsampling/openpathsampling/blob/8eca3946ce68ea44b31709ddb8112812fe24fd79/openpathsampling/experimental/simstore/storable_functions.py#L568-L573

**Comment by dwhswenson** [comment]
> Also, just letting you know that the write access for `openpathsampling` has not carried over to this repo

Access granted, although not as an official "OPS maintainers" but as you individually. This repo is a personal repo, not an official OPS one.

**Pull request merged**
